picoquic/quicctx.c

```diff
@@ -816,7 +816,13 @@ void picoquic_set_default_multipath_option(picoquic_quic_t* quic, int multipath_
 
 void picoquic_set_cwin_max(picoquic_quic_t* quic, uint64_t cwin_max)
 {
-    quic->cwin_max = (cwin_max == 0) ? UINT64_MAX : cwin_max;
+    quic->cwin_max = (cwin_max == 0 || cwin_max < PICOQUIC_MAX_PACKET_SIZE) ? UINT64_MAX : cwin_max;
```

Comment:
```cpp
cwin_max < PICOQUIC_MAX_PACKET_SIZE
```
Is this logic correct?

Reply:
CWIN cannot be less than max packet size.  If it is, then it's invalid just a much as zero is invalid.